source/usage-examples/findOne-output.txt

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Once the comment about using `bson.MarshalExtJSON` is addressed, this file will need to be updated to be:
```
{"title":"The Room","imdb":{"rating":3.5,"votes":25673,"id":368226}}
```

Reply:
Thanks for the note. The result document is updated here: https://github.com/mongodb/docs-golang/pull/2/commits/fd57e936491c7f86224fdd2965cdc78dde59426e